toonz/sources/toonzqt/functiontreeviewer.cpp

```diff
@@ -637,9 +637,9 @@ QString FunctionTreeModel::Channel::getExprRefName() const {
   StageObjectChannelGroup *stageGroup =
       dynamic_cast<StageObjectChannelGroup *>(m_group);
   if (stageGroup) {
-    if (tmpName == "N/S")
+    if (tmpName == "Y")
       tmpName = "ns";
```

Comment:
This function should return the representative expression to refer each channel. It will be used when user middle-drag the column head of the spreadsheet and drop into the expression field.

After enabling the text "y" as an expression referring the Y value, I think it would be better to change to "y" for simplicity.